Classes/extra_functions.py

```diff
@@ -303,6 +304,11 @@ def check(reaction, user, valid_approvers):
     # we aren't going to delete unknown messages. Just react with a question mark.
     await message.add_reaction("\N{BLACK QUESTION MARK ORNAMENT}")
 
-def ts_print(*objects, sep=' ', end='\n', file=sys.stdout, flush=False):
-    timestamp = colored(datetime.now().strftime('%b-%d-%Y %H:%M:%S'), 'green')
-    print(timestamp + *objects, sep=sep, end=end, file=file, flush=flush)
+def ts_print(*objects, sep=' ', end='\n', file=stdout, flush=False):
+    """Adds a colored timestamp to debugging messages in the console"""
+    
+    if objects is None or objects[0] == "":
+        print('')
+        return
```

Comment:
This will fail if objects[0] is "" and then object[1] actually has something you want to print

Reply:
Changed to do the “” check only if length is 1. This is to accommodate “” for newline